[Contributors] Community Members (#10)
wheelnext/wheelnext: docs/who_are_we.md

# Who Are We ?

## Foreword

> We are excited to count among our ranks numerous key contributors to so many key project of the Python Open Source Ecosystem.

Reviewer: ```suggestion
We are excited to count among our ranks numerous key contributors to so many key project of the Python Open Source Ecosystem, and
```

> We are excited to count among our ranks numerous key contributors to so many key project of the Python Open Source Ecosystem, and

Reviewer 2: ```suggestion
We are excited to count among our ranks numerous key contributors to so many key projects of the Python Open Source Ecosystem, and
```

> So many companies / groups / organizations who recognized the value of fixing the issues WheelNext is trying to approach.

Reviewer: ```suggestion
so many companies / groups / organizations who recognized the value of advancing the state of Python packaging through the WheelNext initiative.
```

> Below you will find the list of communities, projects and companies involved in making WheelNext great !

Reviewer 2: ```suggestion
Below you will find the list of communities, projects and companies involved in making WheelNext great!
```

## Open-Source Projects (alphabetic order)

- AstroPy: https://www.astropy.org/

> - Conda: <https://docs.conda.io/projects/conda/en/stable/>

Reviewer 2: And should probably point to conda.org
```suggestion
- Conda: https://conda.org
```

> - Conda: <https://docs.conda.io/projects/conda/en/stable/>

Author: @jezdez if it helps - I can absolutely remove Conda / Anaconda name from this PR and add it back whenever you have approval :)

> - GPU-Mode: <https://github.com/gpu-mode>

Reviewer 3: ```suggestion
- GPU MODE: <https://github.com/gpu-mode>
```

Author: (Noted I'll correct that in a further commit immediately after this one is merged - I don't want to void the approvals here)

- JAX: <https://docs.jax.dev/en/latest/>
- NumFocus: <https://numfocus.org/>
- NumPy: <https://numpy.org/>
- Numba: <https://numba.pydata.org/>
- PyTorch: <https://pytorch.org/>
- RAPIDS: <https://rapids.ai/>

> - Scikit-Build: https://scikit-build.readthedocs.io/en/latest/

Reviewer 4: ```suggestion
- scikit-build: https://scikit-build.readthedocs.io/en/latest/
```

Author: (Noted I'll correct that in a further commit immediately after this one is merged - I don't want to void the approvals here)

> - Scikit-Learn: https://scikit-learn.org/

Reviewer 4: ```suggestion
- scikit-learn: https://scikit-learn.org/
```

Author: (Noted I'll correct that in a further commit immediately after this one is merged - I don't want to void the approvals here)

- Triton: <https://openai.com/index/triton/>

> - UV: <https://docs.astral.sh/uv/>

Reviewer 5: ```suggestion
- uv: <https://docs.astral.sh/uv/>
```

Author: (Noted I'll correct that in a further commit immediately after this one is merged - I don't want to void the approvals here)

- XGBoost: <https://xgboost.readthedocs.io/en/stable/>

## Companies (alphabetic order)

- AMD: <https://www.amd.com/>
- Anaconda: <https://www.anaconda.com/>
- Astral: <https://astral.sh/>
- Google: <https://about.google/>

> - META: <https://about.meta.com/>

Reviewer 4: ```suggestion
- Meta: <https://about.meta.com/>
```

Author: (Noted I'll correct that in a further commit immediately after this one is merged - I don't want to void the approvals here)

- NVIDIA: <https://www.nvidia.com/en-us/>
- OpenAI: <https://openai.com/>
- Probabl.ai: <https://probabl.ai/>

> - QuanSight: <https://quansight.com/>

Reviewer 4: ```suggestion
- Quansight: <https://quansight.com/>
```
